pyrisco/risco.py

```diff
@@ -131,7 +161,10 @@ def __init__(self, raw):
     def partitions(self):
         """Alarm partitions."""
         if self._partitions is None:
-            self._partitions = {p["id"]: Partition(p) for p in self._raw["partitions"]}
+            if self._raw["partitions"] is not None:
```

Comment:
Not sure about this, but maybe better to return an empty collection and have a different property to return the partition-like object for non partition systems? Or maybe rename this property to `panels`?